server/plugin.go

```diff
@@ -455,8 +422,8 @@ func (p *Plugin) handleProfileImage(w http.ResponseWriter, r *http.Request) {
 	}
 	img, err := os.Open(filepath.Join(bundlePath, "assets", "profile.png"))
 	if err != nil {
+		p.API.LogWarn("Cannot read Skype 4 Business plugin profile image", "err", err)
```

Comment:
Why LogWarn instead of LogError, since there is a mlog.Error?

Reply:
I think it's not a critical failure, so I changed it to LogWarn. Do you think LogError is more suitable here?

Reply:
Since the function sole purpose is to handle the profile image, it should return an error if it cannot read it, specially if it is a returning error. I am OK with the change :)